recipes/liblsl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+
+{% set name = "liblsl" %}
+{% set version = "1.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sccn/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7b0c927fba9875c2eb930e5347a339f0aef750291e94045c3f8dbc6e5112d4a5
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('liblsl', max_pin='x') }}
+
+requirements:
+  build:
+    - cmake >=3.12
+    - {{ compiler('cxx') }}
+  host:
+    - pugixml
+
+test:
+  commands:
+    - lslver
+    - test -f $PREFIX/include/lsl_c.h  # [unix]
+    - test -f $PREFIX/include/lsl_cpp.h  # [unix]
+    - if not exist %LIBRARY_INC%\\lsl_c.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\lsl_cpp.h exit 1  # [win]
+    - test -f $PREFIX/lib/liblsl.so  # [linux]
+    - test -f $PREFIX/lib/liblsl.dylib  # [osx]
```

Comment:
```suggestion
    - test -f $PREFIX/lib/liblsl${SHLIB_EXT}  # [unix]
```

Reply:
TIL. I let myself be inspired by the [apache arrow package](https://github.com/conda-forge/arrow-cpp-feedstock/blob/master/recipe/meta.yaml), but that's even better.